app/src/test/java/mozilla/lockbox/presenter/AutofillLockedPresenterTest.kt

```diff
@@ -105,6 +106,6 @@ class AutofillLockedPresenterTest : DisposingTest() {
         Mockito.`when`(fingerprintStore.isKeyguardDeviceSecure).thenReturn(true)
 
         settingStore.unlockWithFingerprintStub.onNext(true)
-        dispatcherObserver.assertValue { it is RouteAction.DialogFragment.FingerprintDialog }
+        dispatcherObserver.assertValue { it is RouteAction.DialogFragment.FingerprintDialog }.await(1, TimeUnit.SECONDS)
```

Comment:
rather than using an await, can you block on the `register` method of the dispatcher? that's usually more stable.